제일-작은-수-제거하기.js

```diff
@@ -0,0 +1,7 @@
+function solution(arr) {
+    arr = arr.filter((item) => item !== Math.min.apply(null, arr));
+
+    arr = arr.length ? arr : [-1];
+
+    return arr;
+}
```

Comment:
```suggestion
function solution(arr) {
    const min = Math.min(...arr);
    arr = arr.filter((item) => item !== min);

    return arr.length ? arr : [-1];;
}
```

Math.min.apply(null, arr)를 filter안에 넣게되면 filter의 반복마다 저 메서드를 계속 수행하기 때문에 복잡도가 증가하게 되서 따로 빼주는게 좋을거같아요.(테스트케이스 1번 수행시간 비교해보시면 차이가 꽤 납니다.) 그리고 aplly(null, arr) 보다는 (...arr) 스프레드 연산자를 쓰는게 더 좋을거같아요.